src/System.Diagnostics.Debug/src/System/Diagnostics/DebuggerStepperBoundaryAttribute.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+namespace System.Diagnostics
+{
+    /// <summary>Indicates the code following the attribute is to be executed in run, not step, mode.</summary>
+    [AttributeUsage(AttributeTargets.Constructor | AttributeTargets.Method, Inherited = false), ComVisible(true)]
```

Comment:
`ComVisible` can be removed. See https://github.com/dotnet/corefx/pull/12113#discussion_r82315784